src/material/chips/chip-grid.spec.ts

```diff
@@ -70,11 +72,13 @@ describe('MDC-based MatChipGrid', () => {
         expect(chips.toArray().every(chip => chip.disabled)).toBe(false);
 
         chipGridInstance.disabled = true;
+        fixture.changeDetectorRef.markForCheck();
```

Comment:
Is this necessary anymore now that you have the subscription to the events and `markForCheck` directly in chip-grid? I guess maybe it is since you're still assigning an input directly.

Reply:
Yeah, this property is a setter, and *does* actually call `markForCheck` on all of the child chip components, but not on itself. At some point in the past we decided to not have all of our inputs call `markForCheck` when they're set programmatically, as that would require turning them all into getter/setters.

I think for now it makes sense to just leave it and call `markForCheck` in the test, but hopefully its something that can be addressed systemically when we migrate to signals